pkg/cosign/tlog.go

```diff
@@ -184,17 +184,13 @@ func doUpload(ctx context.Context, rekorClient *client.Rekor, pe models.Proposed
 	return nil, errors.New("bad response from server")
 }
 
-func rekorEntry(payload, signature, pubKey []byte) hashedrekord_v001.V001Entry {
-	// TODO: Signatures created on a digest using a hash algorithm other than SHA256 will fail
```

Comment:
You should probably leave this TODO here.

Reply:
Done.